Doc/library/sys.rst

```diff
@@ -11,6 +11,39 @@ interpreter and to functions that interact strongly with the interpreter. It is
 always available. Unless explicitly noted otherwise, all variables are read-only.
 
 
+.. data:: abi_info
+
+   An object containing information about the ABI of the currently running
+   Python interpreter.
+   It should include information that affect the CPython ABI in ways that require
+   a specific build of the interpreter chosen from variants that can co-exist on
+   a single machine.  For example, it does not encode the base OS (Linux or
+   Windows), but does include pointer size since some systems support both 32-
+   and 64- builds.  The available entries are the same on all platforms; e.g.
+   pointer_size is available even on 64-bit-only architectures.
+
+   New entries should be added when needed for a supported platform, or (for
+   enabling an unsupported one) by core dev consensus.  Entries should be removed
+   following :pep:`387`.
+
+   The following attributes are available in cpython:
+
+   *pointer_bits* is the width of pointers in bits, as an integer, equivalent
+   to ``8 * sizeof(void *)``, i.e. usually ``32`` or ``64``.
+
+   *free_threaded* is a boolean indicating whether the interpreter was built
+   with the GIL disabled, i.e. with the :option:`--disable-gil` option,
+   aka free-threading.
+
+   *debug* is a boolean indicating whether the interpreter was built in
+   :ref:`debug mode <debug-build>`, i.e. with the :option:`--with-pydebug` option.
+
+   *byteorder* is a string indicating the native byte order, either ``'big'`` or
```

Comment:
I suggest renaming the member to *byte_order* since there are two words: https://en.wiktionary.org/wiki/byte_order

By the way it seems like *endianness* word is more commonly used.

Reply:
I'd keep the name used in [`sys.byteorder`](https://docs.python.org/3/library/sys.html#sys.byteorder) and [`int.from_bytes`](https://docs.python.org/3/library/stdtypes.html#int.from_bytes)/`to_bytes`.

Reply:
I'm reasonably happy with any of the three names. I have commented on the discussion thread to invite other opinions, but if that is ok for you, @vstinner, I would stick with `byteorder`?